### DISCUSSION TITLE
Auto Completion on Managed Packages

### DISCUSSION THREAD

**Discussion by mnunezdm** [forcedotcom/salesforcedx-vscode#5741] [Ideas]
### Summary

When developing in a workspace component, I've told that is a best practice to don't add the namespace suffix to any of data, in order to let the code to be deployed to any place when it's necessary.

The problem is, that doing these, makes it impossible to autocomplete anything

### Steps To Reproduce:

1.  Initialize a new project with a namespace assigned
1.  Create a class
1.  Try to reference it

### Expected result

Autocompletion should work for both cases, trying to reference it with the namespace and without it

### Actual result

Autocompletion is not working for that class, instead, you need to add the namespace.class to get it working

### Additional information

![testAutocompletion](https://user-images.githubusercontent.com/10410852/54775485-6a688a80-4c0e-11e9-9e7e-a49aefbca54e.gif)

**VS Code Version**: 1.32.3

**SFDX CLI Version**: 45.7.0

**OS and version**: Windows 10

**Comment by lcampos** [comment]
Thanks for opening this @mnunezdm . This is a currently a limitation with the Apex language server which we're looking into.
cc @ntotten

**Comment by praksb** [comment]
@mnunezdm I agree this is an annoying bug. While we are working on making this better, as a workaround you could  temporarily set 
"namespace": "" 
in your sfdx-project.json file.

**Comment by Szandor72** [comment]
I had a feature request in that direction: ignore namespace configured in project.json for auto completion. but cannot find it anymore for the love of god. 
Thanks for the workaround!
